mosaicforecast/Mutect_to_MF.sh

```diff
@@ -1,18 +1,22 @@
 ### Code to convert Mutect vcf output to input for MosaicForecast. Adapted from MosaicForecast Snakemake file (https://github.com/parklab/MosaicForecast/blob/master/Snakemake/Snakefile)
```

Comment:
If this script stays this short, you could just put it directly into the command block of the WDL so that you don't have to add it to the docker image